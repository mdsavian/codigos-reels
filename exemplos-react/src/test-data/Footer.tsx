import "./styles.css";

const Footer: React.FC = () => {

    const anoCorrente = new Date().getFullYear()

    return (
        <div>
            <p className="text"> {anoCorrente} </p>            
        </div>
    );
};

export default Footer