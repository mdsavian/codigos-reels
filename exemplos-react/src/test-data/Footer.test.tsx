import { render } from "@testing-library/react"
import Footer from "./Footer"

describe("Footer", ()=>{

    it("should match snapshot", ()=>{

         jest.useFakeTimers()
         jest.setSystemTime(new Date("2022/04/24"))

        const {container} = render(<Footer/>)

        expect(container).toMatchSnapshot()

         jest.useRealTimers()
    })
})