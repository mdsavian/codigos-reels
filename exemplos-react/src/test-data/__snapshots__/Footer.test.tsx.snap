// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<div>
  <div>
    <p
      class="text"
    >
       
      2022
       
    </p>
  </div>
</div>
`;
